### PR TITLE
resloved issue 2608 - Printing to PDF with the page title

### PR DESCRIPTION
**Pull request by TARPTAEYA**

### CODE REVIEW

**Comment by Martii** [comment]
Ref backlink to #2608

**Comment by nowrep** [comment]
This is still using url and not page title as requested in #2608

**Comment by TARPTAEYA** [comment]
fixed

**Comment by nowrep** [comment]
What name it uses when title is `foo/bar/123\456`?

**Comment by TARPTAEYA** [comment]
done

File: src/lib/webengine/webview.cpp [hunk 1]
```diff
@@ -392,14 +392,16 @@ void WebView::printPage()
 {
     Q_ASSERT(m_page);
 
-    QPrintDialog* dialog = new QPrintDialog(this);
+    QPrinter* printer = new QPrinter(QPrinter::HighResolution);
```
Comment (nowrep) [inline src/lib/webengine/webview.cpp]: Any reason for `QPrinter::HighResolution` instead of default?

Comment (TARPTAEYA) [reply src/lib/webengine/webview.cpp]: No specific reason, reverted it back to default.

**Pull request merged**